Created component + styling for the burger station. The images need t…

akhilkasturi opened the pull request:
…o be adjusted to be side view, but this implementation should work for stacking ingredients. Still need to set up communication with backend: If anyone has a preferred way of doing this let me know.

akhilkasturi commented:
I'm planning on creating side view images for some of the ingredients that we cannot reuse the menu buttons for

Andriy-Luchko reviewed (changes requested):
Looks good in terms of functionality. 
Once you finish the side view for all items it'll look sick. 

Suggestion: stack them a bit more since the items do feel like they are floating right now.

Requested change:
<img width="702" alt="Screenshot 2025-03-11 at 11 52 38 PM" src="https://github.com/user-attachments/assets/9f67f536-bb57-45e3-8cbb-186fb2bee1e1" />
Give the boxes minimum sizes/wrap like the AAC board currently has, otherwise it gets funky on different aspect ratios/resolutions.

Andriy-Luchko commented:
Also run precommit before committing so it passes checks.

akhilkasturi commented:
Added minimum sizes for buttons, as well as wrapping for different aspect ratios. Fixed my precommit as well.

akhilkasturi commented:
Updated burger images (some ingredients still messed up), Added clear plate button, added drink component: Employee can add color layers to the cup, add ice (Ice display is temporary i want to change it before the demo), and clear the cup. Added side station, where employee can place a potato on the table, chop it, and place it in the fryer.

Andriy-Luchko reviewed (approved):
Good job, thank you
